python/raydp/torch/estimator.py

```diff
@@ -119,6 +122,14 @@ def __init__(self,
                Note: Now the value can only be False
         :param num_processes_for_data_loader: the number of processes use to speed up data loading
         :param callbacks: which will be executed during training.
+        :param metrics_name: the name of metrics' classes used to evaluate model. the name can be
+               used refer: https://torchmetrics.readthedocs.io/en/latest/, For example:
+               for classification tasks, it can be "Accuracy", "Precision" and "Recall";
+               for regression tasks, it can be "MeanAbsoluteError" or "MeanSquaredError".
+               You can also pass a custom torchmetrics.Metric, the class should be defined
+               with reference to https://torchmetrics.readthedocs.io/en/latest/pages/implement.html.
+        :param metrics_config: the config can be set to metrics which you need to set. Its format
```

Comment:
Maybe simplify "the config can be set to metrics which you need to set." to something like "the optional config for the metrics."